kingfisher_scrapy/spiders/europe_dynamic_base.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+from urllib.parse import urlsplit
+
+import scrapy
+
+from kingfisher_scrapy.base_spider import CompressedFileSpider
+from kingfisher_scrapy.util import components, handle_http_error, join
+
+
+class EuropeDynamicBase(CompressedFileSpider):
```

Comment:
```suggestion
class EuropeanDynamicsBase(CompressedFileSpider):
```

And could you add some documentation similar to the MexicoINAI base class https://github.com/open-contracting/kingfisher-collect/blob/2fbbd6361a0ec959e0603343a4b363f97fae3815/kingfisher_scrapy/spiders/mexico_inai_base.py#L8-L30 ? That way it will be easier for other developers to re use the base class if we need to add a new sub class in the future